src/sentry/web/frontend/generic.py

```diff
@@ -33,6 +33,23 @@ def resolve(path):
     return os.path.split(absolute_path)
 
 
+def static_media_with_manifest(request, **kwargs):
```

Comment:
Would this be used only in dev environments? Or is this for production as well?

Reply:
This *can* be used for a prod environment if django is serving your assets (e.g. ST/onprem), it won't be used in our prod environment as we use a CDN